blueman/main/DBusProxies.py

```diff
@@ -40,11 +40,13 @@ def __init__(self, name, interface_name, object_path='/', systembus=False, *args
 
 class Mechanism(ProxyBase):
     def __init__(self, *args, **kwargs):
-        super().__init__(name='org.blueman.Mechanism', interface_name='org.blueman.Mechanism',
-                         object_path="/org/blueman/mechanism", systembus=True, *args, **kwargs)
+        kwargs.update(name='org.blueman.Mechanism', interface_name='org.blueman.Mechanism',
+                      object_path="/org/blueman/mechanism", systembus=True)
```

Comment:
mypy reports "multiple values for keyword" for the previous code and it does that even if our keyword arguments get passed after `**kwargs`, so that they override caller-defined values. See some discussion in https://github.com/python/mypy/issues/2582.

Reply:
The only reason I added **kwargs is so that we can set additional GObject properties if needed. At the moment we do not make use of this so for these classes we can drop both *args and **kwargs. When we have an actual need we can revisit this and perhaps by then mypy can handle this.